api/routes/companies.js

```diff
@@ -0,0 +1,25 @@
+// const router = require('express').Router()
+// const Company = require('../models/companies')
+
+// // GET Companies 
+// // GET http://localhost:5000/api/companies
+// // Return all companies with all their employees information. Do not return an unit information.
+
+const router = require('express').Router();
+const Unit = require('../models/units');
+
+router.get('/', async (req, res, next) => {
+    const status = 200;
+    const query = req.query.name;
+    try {
+        const companies = await Unit.find(query);
```

Comment:
This should fail, given that `query` at this point should be a string as opposed to an object.

Reply:
Additionally, the query params given in the exercise do not match up with fields on the Company, so something like `employees_gte` will not find anything.